cleanlab/filter.py

```diff
@@ -132,6 +132,8 @@ class 0, 1, ..., K-1.
     filter_by : {'prune_by_class', 'prune_by_noise_rate', 'both', 'confident_learning', 'predicted_neq_given'}, default='prune_by_noise_rate'
       Method to determine which examples are flagged as having label issue, so you can filter/prune them from the dataset. Options:
 
+      - ``'low_normalized_margin'``: filters the examples with *smallest* normalized margin label quality score. The number of issues returned matches :py:func:`count.num_label_issues <cleanlab.count.num_label_issues>`.
+      - ``'low_self_confidence'``: filters the examples with *smallest* self confidence label quality score. The number of issues returned matches :py:func:`count.num_label_issues <cleanlab.count.num_label_issues>`.
```

Comment:
Y/N: Do you want to allow somebody to do: 

```
find_label_issues(filter_by="low_self_confidence", return_indices_ranked_by="normalized_margin")
```
?

Reply:
Currently if somebody does: `find_label_issues(filter_by="low_self_confidence", return_indices_ranked_by="self_confidence")` it seems like there'll be some redundant computation, but I guess we don't care too much because the extra return_indices_ranked_by computation is only over the flagged label issues.

Reply:
> Y/N: Do you want to allow somebody to do:
> 
> ```
> find_label_issues(filter_by="low_self_confidence", return_indices_ranked_by="normalized_margin")
> ```
> 
> ?


Yes. It returns a difference answer and both answers are reasonable that a user might compute them (they might trust filtering done by `self_confidence` more and ranking done by `norm margin` more, so we should keep it this way and let them support both.

The other benefit of keeping the API this way is now the way it works for these two methods is the same as how it works for all 7 methods -- consistency is better.

Should def keep as this API.